subprojects/docs/src/docs/userguide/publishingIvy.adoc

```diff
@@ -107,11 +107,19 @@ For each custom artifact, it is possible to specify the `name`, `extension`, `ty
 Configure custom artifacts as follows:
 
 ++++
-<sample dir="ivy-publish/java-multi-project" id="publishing_ivy:publish-custom-artifact-snippet" title="Publishing additional artifact to Ivy">
+<sample dir="ivy-publish/java-multi-project" id="publishing_ivy:publish-custom-artifact-snippet" title="Adding additional archive artifact to an IvyPublication">
     <sourcefile file="build.gradle" snippet="publish-custom-artifact"/>
 </sample>
 ++++
 
+In addition, instances of api:org.gradle.api.artifacts.PublishArtifact[] can be used. For example, let's assume you have a custom `rpm` task that produces an RPM package of your application and writes it to `rpmFile`. The following sample demonstrates how to create a `PublishArtifact` and specify it as part of a publication:
```

Comment:
"instances of `PublishArtifact` can be used." <- for what? In place of what? It's not clear why the user should care about `PublishArtifact` instances. I also don't see a reference to `PublishArtifact` in the example that follows this paragraph. Which bit of the example is actually creating the `PublishArtifact` instance?